tests/internal/crashtracker/test_crashtracker.py

```diff
@@ -274,3 +275,141 @@ def test_crashtracker_raise_sigbus():
     data = utils.conn_to_bytes(conn)
     conn.close()
     assert b"os_kill" in data
+
+
+@pytest.mark.skipif(not sys.platform.startswith("linux"), reason="Linux only")
+@pytest.mark.subprocess(
+    env={
+        "TEST_FILE_ROOT": os.path.dirname(__file__),
+    }
+)
+def test_crashtracker_preload_default():
+    import os
+    import sys
+
+    import tests.internal.crashtracker.utils as utils
+    from tests.utils import call_program
+
+    # Setup the listening socket before we open ddtrace
+    port, sock = utils.crashtracker_receiver_bind()
+    assert sock
+    os.putenv("DD_TRACE_AGENT_URL", "http://localhost:%d" % port)
+
+    # Call the program
+    root_dir = os.environ["TEST_FILE_ROOT"]
+    stdout, stderr, exitcode, _ = call_program(
+        "ddtrace-run", sys.executable, os.path.join(root_dir, "simple_crashing_preload.py")
+    )
```

Comment:
```python
def test_crashtracker_preload_default(ddtrace_run_python_code_in_subprocess):
    code = """
import ctypes


ctypes.string_at(0)
exit(-1)
    """

    stdout, stderr, exitcode, _ = ddtrace_run_python_code_in_subprocess(code, env=env)
```

`run_python_code_in_subprocess` and `ddtrace_run_python_code_in_subprocess` might help clean this up a little bit, at least not needing to know current root dir, or creating dummy/test files.


what you have is totally fine, just sharing an alternative possible approach. this is a non-blocking comment/no change is needed.

Reply:
Oh cool, TIL.  Will add a patch for that

Reply:
fixed in [219a3a1](https://github.com/DataDog/dd-trace-py/pull/9865/commits/219a3a13927fe41acca3ff2797a467c7aeb415b2)